hivemind_etl/mediawiki/etl.py

```
@@ -100,7 +100,11 @@ def load(self, documents: list[Document]) -> None:
         ingestion_pipeline = CustomIngestionPipeline(
             self.community_id, collection_name=self.platform_id
         )
-        ingestion_pipeline.run_pipeline(documents)
+        # batch documents into chunks of 1000
+        for i in range(0, len(documents), 1000):
+            logging.info(f"Loading batch {i} of {len(documents)} documents into Qdrant!")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)